tapgarden/interface.go

```diff
@@ -59,43 +59,43 @@ type Planter interface {
 }
 
 // BatchState an enum that represents the various stages of a minting batch.
-type BatchState uint8
+type BatchState uint32
 
 const (
 	// BatchStatePending denotes that the batch is pending and may have
 	// some assets allocated to it.
-	BatchStatePending BatchState = 0
+	BatchStatePending BatchState = iota
```

Comment:
Aren't we also storing this state in the DB? If yes, then we must never use `iota`, otherwise a state inserted between existing ones will break existing DB states.

Reply:
Fair point, I only considered appending states for some reason.